Docker/Dockerfile

```diff
@@ -1,8 +1,8 @@
 FROM ubuntu:22.04
 
 LABEL author="OPERA ADT" \
-      description="RTC beta release R2.1" \
-      version="0.2.1-beta"
+      description="RTC gamma release R3" \
+      version="0.3-beta"
```

Comment:
- Just checking if it is okay to use the term `beta` in the gamma delivery.
- A reminder that we need to change the base docker image mentioned in: (#24). I'll take care of this when working on updating the docker image.